file-service-liquibase/src/main/resources/liquibase/file-service-liquibase-db-changesets/004-add-is-deleted-column-to-content-table.xml

```diff
@@ -7,7 +7,7 @@
     <changeSet id="004-add-is-deleted-column-to-content-table" author="TechPod">
 
         <addColumn tableName="content">
-            <column name="deleted" type="boolean">
+            <column name="deleted" type="boolean" defaultValue="true">
```

Comment:
I would have thought the default should be deleted= true, forcing us to explicitly change it when inserting